packages/serverless-contracts-plugin/package.json

```diff
@@ -68,7 +67,7 @@
     "vitest": "^0.28.0"
   },
   "peerDependencies": {
-    "@swarmion/serverless-contracts": "^0.16.1",
+    "@swarmion/serverless-contracts": "workspace:^",
```

Comment:
I fear that this might not be portable for repositories not using workspaces, and moreover if they use `yarn` or `npm` instead of `pnpm`.

As we always deploy the two packages together, it makes sense to make sure the two dependencies have the same version. SO let's keep it that way.

Why moving it as a `peerDependency`?

Reply:
I believe `workspace:^` is resolved when publishing to npm and is not in the published `package.json` so there shouldn't be an issue
I used it because it seems the version of serverless-contracts in peerDependencies was not updated

It was already as peerDependency. I think we should leave it there as it should not be owned by the plugin

Reply:
Well maybe we should remove the plugin completely, we don't use it and it only makes sense with the Serverless Framework

Reply:
> I believe workspace:^ is resolved when publishing to npm and is not in the published package.json so there shouldn't be an issue

Do you have a link that supports this? I cannot find anything on this.

> Well maybe we should remove the plugin completely, we don't use it and it only makes sense with the Serverless Framework

Interesting take, I kind of forgot this could be possible 😅. Maybe we can do this in an other PR.

Reply:
> Do you have a link that supports this? I cannot find anything on this.

you can find it right here : https://pnpm.io/workspaces#publishing-workspace-packages